web/src/components/pages/contribution/success.tsx

```diff
@@ -149,8 +149,7 @@ function Success({
           <div className="info-card">
             <Localized
               id="help-reach-hours"
-              $hours={10000}
-              $language={getString(locale)}>
+              vars={{hours: 10000, language: getString(locale)}}>
```

Comment:
The new API is more explicit about the variables passed into translations as arguments.